trio_websocket/_impl.py

```diff
@@ -530,6 +541,12 @@ def __init__(self, stream, wsproto, *, path=None,
         self._message_parts = []  # type: List[bytes|str]
         self._max_message_size = max_message_size
         self._reader_running = True
+        if wsproto.client:
+            subprotocols = subprotocols or []
```

Comment:
Assuming it's read only, subprotocols could default to empty list or tuple in the args.